php-practice/section2.php

```diff
@@ -117,6 +139,15 @@ function printNewline() {
 // HINT: Use a foreach loop
 // =======================================================================
 
+$sumArray = array("red", "green", "blue", "yellow"); 
+
+foreach ($sumArray as $value) {
+	array_sum ($sumArray += $sumArray);
+}
```

Comment:
Also, make sure to create a function instead of just writing your code inline with no function.